hasher-matcher-actioner/hmalib/common/mappings.py

```diff
@@ -24,11 +24,35 @@
 from hmalib.indexers.s3_indexers import (
     S3BackedMD5Index,
     S3BackedPDQIndex,
+    S3BackedPDQFlatIndex,
     S3BackedInstrumentedIndexMixin,
 )
 
 # Maps from signal type → index to use for that signal type.
-INDEX_MAPPING: t.Dict[t.Type[SignalType], t.Type[S3BackedInstrumentedIndexMixin]] = {
-    PdqSignal: S3BackedPDQIndex,
-    VideoMD5Signal: S3BackedMD5Index,
+INDEX_MAPPING: t.Dict[
```

Comment:
Can you add a comment here? this is no longer signal_type → index, it is signal_type → [index].

Reply:
Done!